include/customizer/cell_customizer.hpp

```diff
@@ -33,16 +33,42 @@ class CellCustomizer
     void Customize(
         const GraphT &graph, Heap &heap, partition::CellStorage &cells, LevelID level, CellID id)
     {
+        Customize(graph, heap, cells, level, id, [](...) {});
+    }
+
+    template <typename GraphT> void Customize(const GraphT &graph, partition::CellStorage &cells)
+    {
+        Customize(graph,
+                  cells,
+                  [this](const GraphT &graph,
+                         Heap &heap,
+                         partition::CellStorage &cells,
+                         LevelID level,
+                         CellID id) { Customize(graph, heap, cells, level, id); });
+    }
+
+  protected:
+    template <typename GraphT, typename RowFn>
+    void Customize(const GraphT &graph,
```

Comment:
There are too many `Customize` methods now. It is better to rename like `CustomizeAllCells`, `CustomizeCell`, `CustomizeRow`